content/arduino-cloud/02.hardware/06.device-provisioning/content.md

```diff
@@ -79,6 +81,8 @@ After completing these steps your device will connect to your Wi-Fi and you will
 
 ![Attach thing after Bluetooth](assets/attach-thing-to-bluetooth.png)
 
+The BLE interface is automatically turned off at the end of the procedure, and it remains off until the board is reset. More information about this can be found in the [How to set up the Reconfiguration Procedure](#how-to-set-up-the-reconfiguration-procedure) section.
```

Comment:
```suggestion
The Bluetooth LE interface is automatically turned off at the end of the procedure, and it remains off until the board is reset. More information about this can be found in the [How to set up the Reconfiguration Procedure](#how-to-set-up-the-reconfiguration-procedure) section.
```